ml-proto/test/memory.wasm

```diff
@@ -54,8 +66,8 @@
           (break)
         )
         (setlocal 2 (mul.i32 (getlocal 0) (const.i32 4)))
-        (setnears.i32 (getlocal 2) (getlocal 0))
-        (setlocal 1 (getnears.i32 (getlocal 2)))
+        (stores.i32 (getlocal 2) (getlocal 0))
+        (setlocal 1 (loads.i32 (getlocal 2)))
```

Comment:
Is there a way for the test to validate what alignment this implicitly corresponds to? IIRC the design said "natural alignment" when unspecified?